web/src/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import {Link} from "react-router-dom";
+
+const Header = () => {
+    return (
+        <header className="text-gray-600 body-font">
+            <div className="container mx-auto flex flex-wrap p-5 flex-col md:flex-row items-center">
+                <nav className="flex lg:w-2/5 flex-wrap items-center text-base md:ml-auto">
+                    <Link to="/models" className="mr-5 hover:text-gray-900">My Models</Link>
+                    <Link to="/" className="mr-5 hover:text-gray-900">Train Model</Link>
```

Comment:
I don't know if the phrase "Train Model", would be understood by a general user. Should we rename this to something like "Chat" or "Chat with repo"?